crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -32,6 +32,9 @@ use shared::Web3;
 use std::time::{Duration, Instant};
 use web3::types::{TransactionReceipt, U64};
 
+/// Used for calculating the maximum additional tip based on the current value of max_fee_per_gas
+const MAX_FEE_PER_GAS_PERCENT: f64 = 0.05; //5%
```

Comment:
Could we get that as a parameter for the driver?

Personally, I think that 0.03 should be sufficient, because the max_fee_per_gas is usually much higher than then spent base_price. E.g. over here its over 50%: https://etherscan.io/tx/0x0dcbbf13a3577b769f9245095901de04172ce1b5a2b20c4425b33a8c33ab4b90

Reply:
Can we make this configurable so that we can change it without requiring code changes when the network conditions change?